File: app/src/donate/java/com/corphish/nightlight/helpers/CrashlyticsHelper.java
package com.corphish.nightlight.helpers;

import android.content.Context;

import com.crashlytics.android.Crashlytics;
import io.fabric.sdk.android.Fabric;

public class CrashlyticsHelper {
    public static void start(Context context) {
        Fabric.with(context, new Crashlytics());
    }
}

File: app/src/donate/java/com/corphish/nightlight/helpers/CrashlyticsHelper.kt
package com.corphish.nightlight.helpers

import android.content.Context

import com.crashlytics.android.Crashlytics
import io.fabric.sdk.android.Fabric

object CrashlyticsHelper {
    fun start(context: Context) {
        Fabric.with(context, Crashlytics())
    }
}